db/migrations/20221226120613_bookings_table.sql

```diff
@@ -0,0 +1,19 @@
+-- migrate:up
+CREATE TABLE bookings (
+    id INT NOT NULL AUTO_INCREMENT,
+    product_id INT NOT NULL,
+    guest_id INT NOT NULL,
+    guest_num INT NOT NULL,
+    check_in DATE NOT NULL,
+    check_out DATE NOT NULL,
```

Comment:
날짜 데이터를 담는 컬럼이니 date를 붙여주는 것이 어떨까요?
```suggestion
    check_in_date DATE NOT NULL,
    check_out_date DATE NOT NULL,
```